### DISCUSSION TITLE
default shot to none?

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2228] [Q&A]
![Screenshot 2024-02-10 at 14-51-25 Fooocus 2 1 862](https://github.com/lllyasviel/Fooocus/assets/35565479/09f83fdd-589b-4ccf-972c-051387413778)

how can i set this? as i have to keep going in a changing

**Comment by mashb1t** [comment]
https://github.com/lllyasviel/Fooocus/pull/1640, not yet merged
